website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -213,36 +200,84 @@ This property contains all the details of the edited rows. It has the following
 
 #### updatedRowIndices
 
-This binding property displays the index number of the updated row. It contains an array of edited row indices.&#x20;
+This binding property displays the index number of the updated row. It contains an array of edited row indices.
 
-For example, if you updated the second row of a table. Now, when you bind this property into a text widget, the property displays the outcome as follows:&#x20;
+For example, if you update the second and fourth rows of a table, the `updatedRowIndices` property contains the value:
 
-```
+```javascript
 [ 
-1
+  1,
+  3
 ]
 ```
 
 #### updatedRow
 
-This property contains the details of the row that triggered the action (`onSubmit`, `onSave` or `onDiscard`). For example, if you bind this property into a text widget, you get an output something like this:
+This property contains the details of the row that triggered the **onSubmit**, **onSave** or **onDiscard** action. For example, if you bind this property into a text widget, you get an output something like this:
 
-```
+```javascript
 {
   "Name": "Updated Name",
   "Date": "Updated Date",
-  "Status ": "Updated Status",
-  "rowIndex": "Row Index"
+  "Status ": "Updated Status"
 }
 ```
 
 ### Events
 
+These properties can be used run code any time one of the events occurs. Be sure to take a look at the list of [supported actions](/reference/appsmith-framework/widget-actions/) that come built in to Appsmith.
+
 | Events        | Description                                                                                                            |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| **onSubmit**  | Action that gets triggered when the user moves away from editing a cell by pressing the enter key or clicking outside. |
-| **onSave**    | Action that gets triggered when the user clicks the save button for a cell or a new row.                                                       |
-| **onDiscard** | Action that gets triggered when user clicks discard button for a cell or a new row.                                                       |
+| **onSubmit**  | Action that gets triggered when the user moves away from editing a cell by pressing the enter key or clicking outside of the cell. |
+| **onSave**    | Action that gets triggered when the user clicks the save button for a cell or a new row.                               |
```

Comment:
note that onSave and onDiscard is now only available for Adding new rows and no longer available for cell editing. it's only onSubmit.

Reply:
@danciaclara Are they going to remove the **Save/Discard** button column in the table while in **Single row** edit mode?

From what I can see on notion it looks like they plan to keep the onSave stuff -- I think it was my description that was confusing. I updated it so it doesn't mention cells.